src/PointerEvents.ts

```diff
@@ -5,12 +5,10 @@ import { Stage } from './Stage';
 
 const Captures = new Map<number, Shape | Stage>();
 
-export interface KonvaPointerEvent extends KonvaEventObject<'touchstart'> {
+export interface KonvaPointerEvent extends KonvaEventObject<MouseEvent> {
```

Comment:
pointer-events inherit from MouseEvent not TouchEvent

Reply:
I guess should be `PointerEvent`?

Reply:
HA, yes that does make more sense